packages/backend/src/logging/winston.ts

```diff
@@ -122,6 +125,24 @@ export const winstonLogger = winston.createLogger({
     transports,
 });
 
+export const auditLogger = winston.createLogger({
+    levels,
+    level: 'audit',
+    transports,
+});
+
+export const logAuditEvent = (event: AuditLogEvent): void => {
+    auditLogger.log({
```

Comment:
Can't we use `winstonLogger` here? it should support the `audit` level.
If so we can delete the `auditLogger`

Reply:
Testing this now1